docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Containerizing Dell EMC OpenManage Ansible Modules
+Containerized image of Dell EMC OpenManage Ansible Modules can be built using the dockerfile available [here](./Dockerfile). We will be using Docker Engine for containerizing OpenManage Ansible Modules. More information about Docker Engine and install instructions can be found [here](https://docs.docker.com/engine/).
+
+## Build
+Download [Dockerfile](./Dockerfile) to the machine where Docker Engine is installed and run below command. 
+```bash
```

Comment:
Can you provide the short description of the each packages used such as base images and ansible packages used in the `Dockerfile`. ? Also the Disk size required.

Reply:
Base image link is added. Details on OMSDK and ansible packages are already covered in repository readme, hence its redundant to cover it over here.

Reply:
will add the size of image.